CovertInfiltration/Config/XComGameCore.ini

```diff
@@ -35,7 +35,18 @@ OPPORTUNE_MOMENT_2_DETECTION_MODIFIER=0.30
 -GlobalLootCarriers=(CarrierName="AlienLoot", LootReferences[0]=(LootTableName="AlienLoot", ForceLevel=0))
 +GlobalLootCarriers=(CarrierName="AlienLoot", LootReferences[0]=(LootTableName="EarlyAlienLoot", ForceLevel=0), LootReferences[1]=(LootTableName="MidAlienLoot", ForceLevel=6), LootReferences[2]=(LootTableName="LateAlienLoot", ForceLevel=9))
 
-LootTables = ( TableName = "EarlyAlienLoot",Loots[0]=(Chance=100,MinCount=1,MaxCount=1,TemplateName="EleriumCore",RollGroup=1), Loots[1]=(Chance=100,MinCount=2,MaxCount=4,TableRef="AdvTrooperM1_BaseLoot",RollGroup=2), Loots[2]=(Chance=100,MinCount=1,MaxCount=2,TableRef="AdvCaptainM1_BaseLoot",RollGroup=3), Loots[32]=(Chance=100,MinCount=1,MaxCount=2,TableRef="Sectoid_BaseLoot",RollGroup=4) )
-LootTables = ( TableName = "MidAlienLoot",Loots[0]=(Chance=100,MinCount=1,MaxCount=2,TemplateName="EleriumCore",RollGroup=1), Loots[1]=(Chance=100,MinCount=2,MaxCount=3,TableRef="AdvStunLancerM1_BaseLoot",RollGroup=2), Loots[2]=(Chance=100,MinCount=1,MaxCount=2,TableRef="Viper_BaseLoot",RollGroup=3), Loots[3]=(Chance=100,MinCount=1,MaxCount=2,TableRef="AdvMEC_M1_BaseLoot",RollGroup=4) )
-LootTables = ( TableName = "LateAlienLoot",Loots[0]=(Chance=100,MinCount=2,MaxCount=2,TemplateName="EleriumCore",RollGroup=1), Loots[1]=(Chance=100,MinCount=1,MaxCount=3,TableRef="AdvShieldBearerM2_BaseLoot",RollGroup=2), Loots[2]=(Chance=100,MinCount=1,MaxCount=2,TableRef="AdvPurifierM1_BaseLoot",RollGroup=3), Loots[3]=(Chance=100,MinCount=2,MaxCount=1,TableRef="Muton_BaseLoot",RollGroup=4) )
-
++LootTables = ( TableName = "EarlyAlienLoot",Loots[0]=(Chance=100,MinCount=1,MaxCount=1,TemplateName="EleriumCore",RollGroup=1), Loots[1]=(Chance=100,MinCount=2,MaxCount=4,TableRef="AdvTrooperM1_BaseLoot",RollGroup=2), Loots[2]=(Chance=100,MinCount=1,MaxCount=2,TableRef="AdvCaptainM1_BaseLoot",RollGroup=3), Loots[3]=(Chance=100,MinCount=1,MaxCount=2,TableRef="Sectoid_BaseLoot",RollGroup=4) )
```

Comment:
Could these be not a single line? It's a pain to read

Reply:
I didn't know if config files cared about newlines, so I went on the safe side.

Reply:
You just have to use correct syntax when splitting on multiple lines and it will parse them properly. See `MissionDefs.ini` or `Schedules.ini` or the encounters inis for examples